scripts/release/requirements.txt

```diff
@@ -1,5 +1,5 @@
 azureml-assets
-azure-ai-ml==1.7.0
+azure-ai-ml>=1.9.0

```

Comment:
Please remove this. The azure-ai-ml dependency will come via azureml-assets now.
```suggestion
```